src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/cache.sends.304.for.matching.etag/accept/server.rpt

```diff
@@ -41,7 +41,7 @@ write nukleus:begin.ext ${http:beginEx()
                              .header(":status", "200")
                              .header("content-length", "11")
                              .header("date", response1Date)
-                             .header("surrogate-control", "max-age=1+100")
+                             .header("surrogate-control", "max-age=10+100")
```

Comment:
Why does `"max-age=1+10"` need to change to `"max-age=10+10"` ?

Reply:
By mistake

Reply:
It turns out the not by mistake sometimes 1 second is enough to be served from the cache. I was getting sporadic failure.